.github/workflows/test.yml

```diff
@@ -91,5 +91,5 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Lint Javascript + SCSS 👕
-        # TODO: enable --max-warnings=0
```

Comment:
I interpret this comment as "we should fix all linting violations and then add `--max-warnings=0`". This may be low priority, how many linting violations are there?

Reply:
That's a good callout. I will check tomorrow and see where we stand right now (might be at 0!)

Reply:
@jeremiah-clothier so the actual count was one warning. Found a weird case where we were getting a false positive for an unused var. According [to this](https://typescript-eslint.io/rules/no-unused-vars/), we can enable this linting rule instead of the built in one to check for unused things even in types. Will update, and set the config to allow 0 warnings